operations/configuration.md

```diff
@@ -383,7 +391,32 @@ Returns all tax environments supported by the API.
 | `Code` | string | required | Code of the tax rate. |
 | `TaxationCode` | string | required | Code of the taxation. |
 | `Value` | number | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
+| `Strategy` | [Tax Rate Strategy](#tax-rate-strategy) | required | Tax strategy type, e.g. relative or flat. |
+
+#### Tax Rate Strategy
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Discriminator` | [Tax Strategy Discriminator](#tax-strategy-discriminator) | required | If tax rate is flat or relative. |
+| `Value` | object | required | Based on Tax Strategy Discriminator, e.g. [Relative Rate Strategy](#relative-rate-strategy) or [Flat Rate Strategy](#flat-rate-strategy). |
```

Comment:
We should agree on single approach how to document such case (discriminated unions) and use it everywhere. It should be consistent with other cases in the documentation, e.g. here. you have the types available documented together with the options. Here it would mean to first document discriminator and introduce "Flat Rate Strategy Data" and "Relative Rate Strategy Data". Tax rate strategy is the object one level above with Discriminator and strategy data. CC @vitasamek  
![image](https://user-images.githubusercontent.com/435787/82847907-ff40d500-9ef0-11ea-8ba2-75cbc666d784.png)